pom.xml

```diff
@@ -860,60 +861,6 @@
                       <exclude>org.ow2.spec.ee:ow2-jta-1.1-spec</exclude>
                       <exclude>jta</exclude>
 
-                      <!-- activation apis - in jdk 6 -->
-                      <exclude>javax.activation</exclude>
```

Comment:
This block of removals are part of the maven enforcer plugin's configuration.  In the past this made sense because all of these were delivered with the JDK so they should have been excluded.  They were all removed in Java 11 and need to be brought in as external dependencies now so this had to be removed to allow for them to be brought in.